### ISSUE TITLE
Armless people can't use telekinesis

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#32205]
(WEB REPORT BY: cartlord REMOTE: 206.221.180.138:7777)
# Revision
640def376fa86972a10781008b1229fbd59a5072

# Description
If you have no arms, you cannot use telekinesis powers, even though you wouldn't logically need hands to use those powers.

# Steps to Reproduce
-Lose arms
-Get T.K.

# What you Expected
-Throw shit

# What Actually Happened
-No shit

**Comment by Intigracy** [comment]
https://github.com/vgstation-coders/vgstation13/pull/8794

**Comment by Exxion** [comment]
It straight up tells you this in-game

**Comment by Eneocho** [comment]
You need to do the jedi hand movement to move stuff telekinetically, man.

**Comment by Exxion** [comment]
Yes, as the in-game message added in the PR linked above tells you
